frontend/assets/javascripts/src/main.js

```diff
@@ -71,6 +71,10 @@ require([
             throw new Error('failed to get geocountry');
         }
     }).then(responseCountryCode => {
+        /**
+         * TODO: Enable Australian CMP by using
+         * property country instead of isInUsa
+        */
```

Comment:
Is this going to happen in this PR, or a subsequent one?

Reply:
The plan is to do this in a follow-up PR. From the conversations we've had, and testing I've carried out updating the library but continuing to use the old `isInUsa` property causes no issues, so can be merged now.